### DISCUSSION TITLE
Line decorations is not display on minimap

### DISCUSSION THREAD

**Discussion by vUdav** [microsoft/vscode-discussions#2461] [Extension Development QnA] [answered]
I want to add to the extension the display of highlighted lines that would be visible on the minimap. For example, how this is done in VS Code, when errors and warnings are displayed. I couldn't find any suitable solution.

Repo: https://github.com/vUdav/vue-tag-highlight

<img width="878" alt="Снимок экрана 2024-12-24 в 15 59 56" src="https://github.com/user-attachments/assets/8b108445-599a-4ecd-b98d-f0162c32ae09" />

**Comment by alefragnani** [comment, marked as answer]
Hi @vUdav ,

The Minimap does not support the **Decorators API** yet, so any decoration you create won’t be available. 

There is however, an open issue requesting that https://github.com/microsoft/vscode/issues/82808. So I suggest you to upvote and add comments, if you have any additional comment to the request.

Hope this helps.
